src/js/stores/NotificationStore.jsx

```diff
@@ -124,35 +124,34 @@ class NotificationStore extends EventEmitter {
 
   getValid() {
```

Comment:
I just made a change to this function in master to let moment do the work, careful with this merge.